python/dgl/distributed/constants.py

```diff
@@ -2,3 +2,26 @@
 
 # Maximum size of message queue in bytes
 MAX_QUEUE_SIZE = 20*1024*1024*1024
+
+
+def get_type_str(dtype):
```

Comment:
do you need this?
https://github.com/dmlc/dgl/blob/master/python/dgl/backend/pytorch/tensor.py#L19

Reply:
`F.reverse_data_type_dict[dtype]` is probably what you want

Reply:
Thanks!